index.js

```diff
@@ -256,6 +256,29 @@ const execa = (command, args, options) => {
 		return Promise.reject(error);
 	}
 
+	const originalKill = spawned.kill.bind(spawned);
+	spawned.kill = (signal, options = {}) => {
```

Comment:
```suggestion
	spawned.kill = (signal = 'SIGTERM', options = {}) => {
```

Then you can remove `if signal === undefined` below

Reply:
Fixed: 2987b9fbe8cf34d560942360a2c5e095f0b83108